js_modules/dagster-ui/packages/ui-core/src/runs/getRunFilterProviders.ts

```diff
@@ -4,7 +4,17 @@ import uniq from 'lodash/uniq';
 import {DagsterEventType} from '../graphql/types';
 
 const typeValues = memoize(() =>
-  uniq(Object.values(DagsterEventType).map(eventTypeToDisplayType)).sort(),
+  uniq(
+    Object.values(DagsterEventType)
+      .filter(
+        (t) =>
+          ![
+            DagsterEventType.ASSET_CHECK_EVALUATION_PLANNED,
+            DagsterEventType.ASSET_MATERIALIZATION_PLANNED,
+          ].includes(t),
+      )
+      .map(eventTypeToDisplayType),
+  ).sort(),
```

Comment:
🤷 

```
const hiddenTypes = new Set([
  DagsterEventType.ASSET_CHECK_EVALUATION_PLANNED,
  DagsterEventType.ASSET_MATERIALIZATION_PLANNED,
]);

const allValues = new Set(Object.values(DagsterEventType));

const typeValues = memoize(() => {
  return Array.from(allValues.difference(hiddenTypes))
    .map(eventTypeToDisplayType)
    .sort();
});
```

Reply:
why do we even need `memoize` at all here?

Reply:
I like this, seems cleaner to me! Will fix

Reply:
Upon further inspection it looks like we can't use Set.difference just yet, and the memoize call here breaks an import-order issue with `DagsterEventType` not yet being defined when the file is loaded. :( I guess I'll leave it for now and we can clean it up when we move to Set.difference!